README.md

```diff
@@ -1,3 +1,7 @@
+# Hiring Offer
+Hey Pascal, 
+I'm Ack from https://www.camb.ai -- we've developed foundational models in speech and language to create one of the world's most powerful zero-shot dubbing AI in **every** language. We're a VC-backed young team of researchers (all < 25yrs) from Carnegie Mellon, ex-Apple and more. I'd love to chat, your profile and work thus far. Hmu at ack@camb.ai if interested.
+
```

Comment:
Everything is a message board, I guess. At least GitHub already did the captcha for this entry 🤣